### PR TITLE
Feature/wynd dex 

### PR DESCRIPTION
**Pull request by SirTLB**
This PR implements the wynd dex formats to the bot and refactors the code where needed.
- [x] I have read [Migaloo's contribution guidelines](https://github.com/White-Whale-Defi-Platform/migaloo-bots/blob/main/CONTRIBUTING.md).
- [x] My pull request has a sound title and description (not something vague like `Update index.md`)
- [x] All existing and new tests are passing (if any).
- [x] I updated/added relevant documentation.
- [x] The code is formatted properly using eslint

### CODE REVIEW

File: botlog.txt [hunk 1]
```diff
@@ -0,0 +1,103 @@
+
```
Comment (kaimen-sano) [inline botlog.txt:1]: Was this file meant to be in the PR?

File: botlot.txt [hunk 2]
```diff
@@ -0,0 +1,102 @@
+
```
Comment (kaimen-sano) [inline botlot.txt:1]: And this?

File: docs/CODE_OF_CONDUCT [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Code of Conduct
```
Comment (kaimen-sano) [inline docs/CODE_OF_CONDUCT:1]: These files seem to be duplicate of the `.md` files under `docs`?

**Review by kaimen-sano — CHANGES_REQUESTED** [review]
A few basic things here

File: botlog.txt [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+> bot-example@1.0.0 start
+> clear && echo "building..." && npm run build && clear && node out/index.js
+
+[H[2J[3J"building..." 
+
+> bot-example@1.0.0 build
```
Comment (0xFable) [inline botlog.txt:7]: .

**Review by 0xFable — DISMISSED** [review]
Changes related to Wyndex are looking good however I see a few files that probably shouldn't be merged in that @kaimen-sano mentioned, could you either remove them via git or start a new PR if it was some weird merge issues. Happy to help DM me anytime.

File: botlog.txt (hunk 1 above)
Comment (SirTLB) [reply botlog.txt:1]: nope you are right, because I messed up the history hard to follow

File: botlot.txt (hunk 2 above)
Comment (SirTLB) [reply botlot.txt:1]: nope

File: docs/CODE_OF_CONDUCT (hunk 3 above)
Comment (SirTLB) [reply docs/CODE_OF_CONDUCT:1]: nope

File: botlog.txt (hunk 4 above)
Comment (SirTLB) [reply botlog.txt:7]: correct

**Review by SirTLB — COMMENTED** [review]
done
